src/azure-cli/azure/cli/command_modules/sql/custom.py

```diff
@@ -2675,6 +2699,23 @@ def managed_instance_create(
     kwargs['sku'] = _find_managed_instance_sku_from_capabilities(cmd.cli_ctx, kwargs['location'], sku)
     kwargs['subnet_id'] = virtual_network_subnet_id
 
+    if not kwargs['yes'] and kwargs['location'].lower() in ['southeastasia', 'brazilsouth', 'eastasia']:
```

Comment:
it seems you want to prompt custom message during operation, right?

Reply:
Yes, I want to prompt for confirmation, but not always. Just in case location is within the 3 values.

Reply:
I considered using confirmation=True, but since I don't need confirmation every time, it is not usable